solidity/dashboard/src/components/BondingSection.jsx

```diff
@@ -6,6 +6,7 @@ import Button, { SubmitButton } from "./Button"
 import { displayAmount } from "../utils/token.utils"
 import { useModal } from "../hooks/useModal"
 import AddEthModal from "./AddETHModal"
+import WithdrawEthModal from "./WithdrawETHModal"
```

Comment:
Lots of unnecessary imports in the `BondingSection.jsx`.

Reply:
Cleaned up